src/components/Common/Avatar.tsx

```diff
@@ -58,7 +58,9 @@ const Avatar: React.FC<AvatarProps> = ({
   className,
   icon,
 }) => {
-  const [bgColor] = propColors || (name ? toColor(name) : toColor(""));
+  const avatarText = name?.match(/[a-zA-Z]/g)?.join("");
```

Comment:
Why is name optional for Avatar component 🤔, also this will break the patient login page. 
![image](https://github.com/user-attachments/assets/c071fd6c-feba-4c63-a8a8-401ce71f5c5e)


Maybe set a defult if name is not avilable like `User`?